doc/source/raster_vector_point.md

```diff
@@ -0,0 +1,191 @@
+---
+file_format: mystnb
+jupytext:
+  formats: md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+kernelspec:
+  display_name: geoutils-env
+  language: python
+  name: geoutils
+---
+(raster-vector-point)=
+# Raster–vector–point interface
+
+GeoUtils provides functionalities at the interface of rasters, vectors and point clouds, allowing to consistently perform 
+operations such as mask creation or point interpolation **respecting both georeferencing and nodata values**.
+
+```{code-cell} ipython3
+:tags: [remove-cell]
+
+# To get a good resolution for displayed figures
+from matplotlib import pyplot
+pyplot.rcParams['figure.dpi'] = 600
+pyplot.rcParams['savefig.dpi'] = 600
+pyplot.rcParams['font.size'] = 9
+```
+
+## Raster–vector operations
+
+### Rasterize
+
+{func}`geoutils.Vector.rasterize`
+
+Rasterization of a vector is **an operation that allows to translate some information of the vector data into a raster**, by 
+setting the values raster pixels intersecting a vector geometry feature to that of an attribute of the vector 
+associated to the geometry (e.g., feature ID, area or any other value).
+
+Rasterization generally implies some loss of information, as there is no exact way of representing a vector on a grid. 
+Rather, the choice of which pixels are attributed a value depends on the amount of intersection with the vector 
+geometries and so includes several options (percent of area intersected, all touched, etc).
+
+```{code-cell} ipython3
+:tags: [hide-cell]
+:mystnb:
+:  code_prompt_show: "Show the code for opening example files"
+:  code_prompt_hide: "Hide the code for opening example files"
+
+import matplotlib.pyplot as plt
+import geoutils as gu
+import numpy as np
+
+rast = gu.Raster(gu.examples.get_path("everest_landsat_b4"))
+rast.set_nodata(0)  # Annoying to have to do this here, should we update it in the example?
```

Comment:
I would say it's good to stay close to the original data. Landsat images don't come with a proper nodata value.

Reply:
OK leaving like this then. Otherwise default nodata 255 create weird artefacts in the plots.